Directory.Packages.props

```diff
@@ -2,10 +2,10 @@
 
   <ItemGroup>
     <PackageVersion Include="Expecto" Version="[9.0, 10.0)" />
-    <PackageVersion Include="FSharp.Core" Version="[4.6.0,)" />
+    <PackageVersion Include="FSharp.Core" Version="[6.0,)" />
```

Comment:
Expecto depends on 4.6.0 - any good reason to change this here?

Reply:
As long as this library is run by Visual Studio but not client code it thinks that it may not have any bad impact. o you think it can?

Reply:
It's not only run by visual studio. It's run in CI, and on clients using `dotnet test` for instance. The main point is that this forces consumers to upgrade `FSharp.Core`.

Reply:
Forces to do that in test project

Reply:
I agree that we can switch back as it was earlier